acoustid/future/fpindex/updater.py

```
@@ -0,0 +1,204 @@
+                )
+                delay = REPL_MIN_DELAY
+            else:
+                delay = min(delay * 2, REPL_MAX_DELAY)
+
+            # if we processed less than the full batch size, wait a bit
+            if len(changes) < REPL_BATCH_SIZE:
+                logger.info("Waiting %s seconds", delay)
+                try:
+                    await asyncio.wait_for(shutdown_event.wait(), delay)
+                except asyncio.TimeoutError:
+                    pass
```

Ruff (0.8.2)

Use `contextlib.suppress(asyncio.TimeoutError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(asyncio.TimeoutError)`

(SIM105)